05/DB.test.js

```diff
@@ -1 +1,10 @@
-import DB from './DB';
\ No newline at end of file
+import DB from "./DB";
+
+describe("DB", () => {
+  it("ID should be a number", async () => {
+    const database = new DB();
+    const id = await database.insert(data.id);
+
+    expect(typeof id).toBe("number");
```

Comment:
Tutaj konstrukcja jest nieprawidłowa. Powinno byc coś takiego:
```
it("ID should be a number", async () => {
    const database = new DB();
    const data = {name: 'Mateusz'}

    const dataInserted = await database.insert(data);

    expect(typeof dataInserted.id).toBe("number");
  });
```
Do `.insert()` wprowadzamy dane, a funkcja zwraca obiekt z utworzonym `id`.

Jeśli chcemy sprawdzić czy działa sprawdzanie parametru `id` to wtedy musimy obsłużyć błąd.
Trzeba go obsłużyć jak kod asynchroniczny bo tam mamy obietnice (https://jestjs.io/docs/asynchronous)
```
    expect.assertions(1);

    const database = new DB();
    const dataWithId = {name: 'Mateusz', id: 'string'}

    try {
      await database.insert(dataWithId)
    } catch (e) {
      expect(e).toMatch('ID can be only number!');
    }
```

Reply:
Muszę się jeszcze lepiej w takim razie z tą dokumentacją zapoznać jeszcze. :)